app/controllers/posts_controller.rb

```diff
@@ -63,6 +65,12 @@ def destroy
 
   def status
     @status = PostStatus.new(@post)
+    respond_to do |format|
+      format.html do
+        redirect_to @post if @status.completed?
+      end
+      format.json {}
```

Comment:
This is still returning the jbuilder template, but it looks like it's returning an empty hash (which it can't be for obvious reasons). Is this line even needed or will Rails just "do its thing" when asked for the json format?